### PR TITLE
[3.3.3]lwm2m_trust_cert

### PR DESCRIPTION
**Pull request by nickAS21**
- lwm2m delete security files jks and in yml security enable = false

### CODE REVIEW

File: application/src/test/java/org/thingsboard/server/transport/lwm2m/AbstractLwM2MIntegrationTest.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
 package org.thingsboard.server.transport.lwm2m;
 
 import com.fasterxml.jackson.core.type.TypeReference;
+import lombok.extern.slf4j.Slf4j;
```
Comment (YevhenBondarenko) [inline application/src/test/java/org/thingsboard/server/transport/lwm2m/AbstractLwM2MIntegrationTest.java:19]: not used

File: application/src/test/java/org/thingsboard/server/controller/TbTestWebSocketClient.java [hunk 2]
```diff
@@ -74,7 +74,7 @@ public void send(String text) throws NotYetConnectedException {
     }
 
     public String waitForUpdate() {
-        return waitForUpdate(TimeUnit.SECONDS.toMillis(3));
+        return waitForUpdate(TimeUnit.SECONDS.toMillis(8));
```
Comment (YevhenBondarenko) [inline application/src/test/java/org/thingsboard/server/controller/TbTestWebSocketClient.java:77]: why?

File: application/src/test/java/org/thingsboard/server/transport/lwm2m/Lwm2mTestHelper.java [hunk 3]
```diff
@@ -55,6 +53,7 @@
     public static final int resourceId_2 = 2;
     public static final int resourceId_3 = 3;
     public static final int resourceId_4 = 4;
+    public static final int resourceId_7 = 7;
```
Comment (YevhenBondarenko) [inline application/src/test/java/org/thingsboard/server/transport/lwm2m/Lwm2mTestHelper.java:56]: should look like RESOURCE_ID_7 or RESOURCE_7

File: application/src/test/java/org/thingsboard/server/transport/lwm2m/ota/sql/OtaLwM2MIntegrationTest.java [hunk 4]
```diff
@@ -204,12 +200,12 @@ public void testFirmwareUpdateByObject5() throws Exception {
      * */
     @Test
     public void testSoftwareUpdateByObject9() throws Exception {
-        String endpoint = "Ota9_Device";
-        setEndpoint(endpoint);
+//        String endpoint = "Ota9_Device";
```
Comment (YevhenBondarenko) [inline application/src/test/java/org/thingsboard/server/transport/lwm2m/ota/sql/OtaLwM2MIntegrationTest.java:203]: remove

File: application/src/test/resources/lwm2m/credentials/shell/lwm2m_cfssl_chain_all_for_test.sh [hunk 5]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
```
Comment (YevhenBondarenko) [inline application/src/test/resources/lwm2m/credentials/shell/lwm2m_cfssl_chain_all_for_test.sh:1]: this script is really needed for tests?

File: application/src/test/resources/lwm2m/credentials/shell/lwm2m_cfssl_chain_server_for_test.sh [hunk 6]
```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env bash
```
Comment (YevhenBondarenko) [inline application/src/test/resources/lwm2m/credentials/shell/lwm2m_cfssl_chain_server_for_test.sh:1]: this script is really needed for tests?

File: application/src/test/java/org/thingsboard/server/transport/lwm2m/AbstractLwM2MIntegrationTest.java (hunk 1 above)
Comment (nickAS21) [reply application/src/test/java/org/thingsboard/server/transport/lwm2m/AbstractLwM2MIntegrationTest.java:19]: Ок

File: application/src/test/java/org/thingsboard/server/transport/lwm2m/Lwm2mTestHelper.java (hunk 3 above)
Comment (nickAS21) [reply application/src/test/java/org/thingsboard/server/transport/lwm2m/Lwm2mTestHelper.java:56]: RESOURCE_ID_7

File: application/src/test/java/org/thingsboard/server/transport/lwm2m/ota/sql/OtaLwM2MIntegrationTest.java (hunk 4 above)
Comment (nickAS21) [reply application/src/test/java/org/thingsboard/server/transport/lwm2m/ota/sql/OtaLwM2MIntegrationTest.java:203]: Ок

File: application/src/test/resources/lwm2m/credentials/shell/lwm2m_cfssl_chain_all_for_test.sh (hunk 5 above)
Comment (nickAS21) [reply application/src/test/resources/lwm2m/credentials/shell/lwm2m_cfssl_chain_all_for_test.sh:1]: Yes

File: application/src/test/resources/lwm2m/credentials/shell/lwm2m_cfssl_chain_server_for_test.sh (hunk 6 above)
Comment (nickAS21) [reply application/src/test/resources/lwm2m/credentials/shell/lwm2m_cfssl_chain_server_for_test.sh:1]: Yes

File: application/src/test/java/org/thingsboard/server/controller/TbTestWebSocketClient.java (hunk 2 above)
Comment (nickAS21) [reply application/src/test/java/org/thingsboard/server/controller/TbTestWebSocketClient.java]: Ok

**Pull request merged**